lib/ssl/src/ssl_manager.erl

```diff
@@ -465,7 +465,7 @@ invalidate_session(Cache, CacheCb, Key, _Session,
 
 clean_cert_db(Ref, CertDb, RefDb, FileMapDb, File) ->
     case ssl_pkix_db:ref_count(Ref, RefDb, 0) of
-	0 ->	  
```

Comment:
I think that it is nice to make white space changes as a separate commit for cosmetic enhancments so that it is easy to see what code is important to the code logic. This is however such a small change that it perhaps matters less.

Reply:
Yes I agree in general. Will avoid that in future.